erpnext/accounts/doctype/opening_invoice_creation_tool/opening_invoice_creation_tool.py

```diff
@@ -213,6 +213,8 @@ def start_import(invoices):
 	if errors:
 		frappe.msgprint(_("You had {} errors while creating opening invoices. Check {} for more details")
 			.format(errors, "<a href='/app/List/Error Log' class='variant-click'>Error Log</a>"), indicator="red", title=_("Error Occured"))
+	if len(invoices) < 5:
```

Comment:
I guess the screen is already frozen while the invoices are being created. If the invoices are not created via a background job then the invoice names are returned to the client-side so this message can be printed there if invoice names are found